dbt_project.yml

```diff
@@ -1,21 +1,21 @@
-
 # Name your project! Project names should contain only lowercase characters
 # and underscores. A good package name should reflect your organization's
 # name or the intended use of these models
-name: 'ro_exemplar'
+name: 'rapid_onboarding_exemplar'
 version: '1.0.0'
 config-version: 2
+require-dbt-version: '>=1.0.0'
 
 # This setting configures which "profile" dbt uses for this project.
 profile: 'default'
 
 # These configurations specify where dbt should look for different types of files.
-# The `source-paths` config, for example, states that models in this project can be
+# The `model-paths` config, for example, states that models in this project can be
 # found in the "models/" directory. You probably won't need to change these!
-source-paths: ["models"]
+model-paths: ["models"]
 analysis-paths: ["analysis"]
 test-paths: ["tests"]
-data-paths: ["data"]
+seed-paths: ["data"]
```

Comment:
Should we change this folder to be named "seeds" to be aligned with our [seeds docs](https://docs.getdbt.com/docs/building-a-dbt-project/seeds)?

Reply:
Good eyes @graciegoheen ! Just updated